OracleUnifiedDirectorySM/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
```

Comment:
This file should have been ignored via `.gitignore`. I recommend removing it completely from your fork.